protos/dcs/coalition/v0/coalition.proto

```diff
@@ -77,25 +77,26 @@ message AddGroupRequest {
       ALTITUDE_TYPE_RADIO = 2;
     }
 
-    enum Type {
-      TYPE_UNSPECIFIED = 0;
-      TAKEOFF = 1;
-      TAKEOFF_PARKING = 2;
-      TURNING_POINT = 3;
-      TAKEOFF_PARKING_HOT = 4;
-      LAND = 5;
+    enum PointType {
+      POINT_TYPE_UNSPECIFIED = 0;
+      POINT_TYPE_TAKEOFF = 1;
+      POINT_TYPE_TAKEOFF_PARKING = 2;
+      POINT_TYPE_TURNING_POINT = 3;
+      POINT_TYPE_TAKEOFF_PARKING_HOT = 4;
+      POINT_TYPE_LAND = 5;
```

Comment:
This only applies to planes. I don't think this should be part of the `Point` type. 

We should uplift the starting position and configuration to the various `xxxxGroupTemplate` and append waypoints.

Reply:
See https://github.com/DCS-gRPC/rust-server/issues/76#issuecomment-973463493 for ideas around this.